brave/brave-core
# Adds events to internals page

Resolves https://github.com/brave/brave-browser/issues/10781

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- go to internals page
- go to event logs tab
- make sure that you see data in there
- go to rewards page
- disable AC
- go back to internals page
- make sure that you see new records saying that ac is 0

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
